.vscode/settings.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
Could we remove this file from the commit? Clashing with my color vibes. We can add this to `.gitignore` so everyone can use their own

Reply:
Didn't even notice it got added here. Will do.